backend/experiment/rules/toontjehogerkids_1_mozart.py

```diff
@@ -60,13 +60,15 @@ def get_final_round(self, session):
         )
 
         # Info page
+        debrief_message = "Merkte je dat de puzzel beter ging na het \
+            horen van Mozart? Dit noemen we het 'Mozart effect'! Wil je meer weten over het Mozart effect? Bekijk dan de filmpjes!"
         body = render_to_string(
-            join('info', 'toontjehoger', 'experiment1.html'))
+            join('info', 'toontjehogerkids', 'debrief.html'), {'debrief': debrief_message, 'vid1': "https://www.youtube.com/embed/iPnZOrPPcBM?si=hFwmD4bVKGW2eGs1", 'vid2': "https://www.youtube.com/embed/pqAt-dvAqoI?si=z06Bd5UBVaA_go3w"})
         info = Info(
             body=body,
             heading="Het Mozart effect",
             button_label="Terug naar ToontjeHoger",
-            button_link="/toontjehoger"
+            button_link="/collection/thkids"
```

Comment:
Although good for now and keeping in mind that we have a deadline to make, perhaps we can get the value of this variable by checking if the current experiment is part of a collection, or the collection that is related to the current session (if that's possible).

Reply:
Yes, I thought the same. Hard-coding here for now, but eventually, would like to have a utility function to pull out the adequate information from the session data.